regression-test/package.json

```diff
@@ -33,6 +33,7 @@
     "webpack-cli": "^4.9.2"
   },
   "dependencies": {
+    "@instructure/ui": "8.24.2",
     "react": "^17.0.2",
```

Comment:
I think this is generated by a script, you should check with @Brailor whether its needed here.

Reply:
Yeah, he told me to delete it :) Thanks!